Tasks/JavaToolInstallerV0/javatoolinstaller.ts

```diff
@@ -43,6 +45,13 @@ async function getJava(versionSpec: string) {
 
     if (version) { //This version of Java JDK is already in the cache. Use it instead of downloading again.
         console.log(taskLib.loc('Info_ResolvedToolFromCache', version));
+    } else if (preInstalled) {
+        toolLib.debug("Use preinstalled java");
```

Comment:
Could you make this a log line instead of debug and have it print out the location of the pre-installed Java JDK so the user can see where we are looking for it

Reply:
Solved in https://github.com/microsoft/azure-pipelines-tasks/pull/12918/commits/23ddb2333219736f734534a6012f0503da057b49